### PR TITLE
[DOC-11822] Document ANALYTICS() built-in function

### PR DESCRIPTION
**Pull request by julia-cb**
Doc preview: [Eventing Language Constructs - Staging](https://staging.docs-test.couchbase.com/server/7.6/eventing/eventing-language-constructs.html#analytics_call)

Jira ticket: [DOC-11822](https://issues.couchbase.com/browse/DOC-11822)

Changes to the [Eventing Language Constructs](https://docs.couchbase.com/server/current/eventing/eventing-language-constructs.html) page:

- Added new section ANALYTICS() Function Call to Built-in Functions
- Updated language and structure throughout page
- Changed "removed features" to "unsupported features"
- Changed "added features" to "supported features"

### CODE REVIEW

File: modules/eventing/pages/eventing-language-constructs.adoc [hunk 1]
```diff
@@ -382,49 +346,102 @@ _Example iterator using a named params object_
...
+|Exceptions thrown
+|The `N1QL()` function throws an exception if the underlying {sqlpp} query fails to parse or does not start to execute.
+
+The returned iterable handle throws an exception if the underlying {sqlpp} query fails after it has started.
+
+The `close()` method on the iterable handle can throw an exception if the underlying {sqlpp} query cancellation finds an unexpected error.
+
+|===
+
+[#analytics_call]
```
Comment (julia-cb) [inline modules/eventing/pages/eventing-language-constructs.adoc:376]: Hi @jon-strabala, this section on the ANALYTICS() Function Call is the only new content on the page. Everything else is just reworked existing content.

File: modules/eventing/pages/eventing-language-constructs.adoc [hunk 2]
```diff
@@ -382,49 +346,106 @@ _Example iterator using a named params object_
...
 
-* _Exceptions Thrown_
-+
-The N1QL() function throws an exception if the underlying {sqlpp} query fails to parse or start executing. The returned Iterable handler throws an exception if the underlying {sqlpp} query fails after starting. The close() method on the iterable handle can throw an exception if underlying {sqlpp} query cancellation encounters an unexpected error.
+    // Log the KEY, AIRLINE and ROUTE_CNT
+    log("key: " + meta.id + ", airline: " + doc.iata + ", route_cnt: " + route_cnt);
+}
+----
 
+For more information about {sqlpp} Analytics, see the xref:server/analytics:1_intro.adoc[{sqlpp} Analytics reference guide].
```
Comment (julia-cb) [inline modules/eventing/pages/eventing-language-constructs.adoc:422]: New content about the ANALYTICS() function call.

File: modules/eventing/pages/eventing-language-constructs.adoc [hunk 3]
```diff
@@ -382,49 +346,106 @@ _Example iterator using a named params object_
...
+|===
+
+[#analytics_call]
+=== `ANALYTICS()` Function Call
+
+ifeval::['{page-component-version}' == '7.6'] 
+_(Introduced in Couchbase Server 7.6)_ 
+endif::[]
+
+The `ANALYTICS()` function provides integration with {sqlpp} Analytics directly form the Eventing Service.
```
Comment (tejaskumars11) [inline modules/eventing/pages/eventing-language-constructs.adoc]: A small typo needs a change.
form -> from

Comment (julia-cb) [reply modules/eventing/pages/eventing-language-constructs.adoc]: Ohh good catch! I'll fix it. Thanks Tejas :)

**Comment by julia-cb** [comment]
@jon-strabala Hi Jon, I'll go ahead and merge this PR for 7.6. I'll revisit it if changes are needed in the future.

**Pull request merged**